rj_gameplay/rj_gameplay/play/basic122.py

```diff
@@ -48,8 +48,8 @@ def tick(
         world_state: rc.WorldState,
         prev_results: role.assignment.FlatRoleResults,
         props,
-    ) -> Tuple[Dict[Type[tactic.SkillEntry], List[role.RoleRequest]],
-               List[tactic.SkillEntry]]:
+    ) -> Tuple[Dict[tactic.SkillEntry,
+                    List[role.RoleResult]], List[tactic.SkillEntry]]:
```

Comment:
Function "tick" had incorrect return type. Fixed to match its interface "IPlay" definition.

Reply:
Not too related to this particular change but it would be more readable to make a NamedTuple or something to give semantic meaning to this type

Reply:
```suggestion
                    List[role.RoleResult]], List[tactic.SkillEntry]]: # TODO: create NamedTuple for this type
```